Doc/library/socket.rst

```diff
@@ -595,6 +595,65 @@ The following functions all create :ref:`socket objects <socket-objects>`.
    .. versionchanged:: 3.2
       *source_address* was added.
 
+.. function:: bind_socket(address, *, family=AF_UNSPEC, type=SOCK_STREAM, backlog=128, reuse_addr=None, reuse_port=False, flags=None, hybrid_ipv46=False)
```

Comment:
bikeshedding on the function name... since listen can also be involved, maybe reuse asyncio name: "create_server()"?

In asyncio, it's more complex than "just" create a socket + bind + listen, the "server" really handles incoming connections, etc.

But I still like "server" name since I'm always confused by the socket API, I never recall how to use it, and "server" is an obvious hint and it can be used to listen for incoming *client* connections.

Well, it's just a suggestion :-)